Makefile

```diff
@@ -1,7 +1,7 @@
 all: binary test
 
 binary:
-	go build github.com/docker/cnab-to-oci/cmd/cnab-to-oci
```

Comment:
In a follow-up, maybe use something like this https://github.com/docker/compose-on-kubernetes/blob/master/Makefile#L47 ?

Reply:
Done (in a very simplistic way, just for it's needs at this point)